samples/extension-module/dummy/DummyApi.ts

```diff
@@ -1,3 +1,4 @@
+import type { Query } from '../../../../aries-framework-javascript/packages/core/src/storage/StorageService'
```

Comment:
This import should not depend on the directory the repo is cloned

Reply:
On it